go/mysql/flavor.go

```diff
@@ -52,6 +52,7 @@ const (
 	MySQLJSONFlavorCapability
 	MySQLUpgradeInServerFlavorCapability
 	DynamicRedoLogCapacityFlavorCapability // supported in MySQL 8.0.30 and above: https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-30.html
+	DisableRedoLogCapacityFlavorCapability // supported in MySQL 8.0.21 and above: https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-21.html
```

Comment:
Sorry, I totally missed the name earlier. This should be: `DisableRedoLogFlavorCapability`. It doesn't have anything to do with the capacity.